app/models/comment.rb

```diff
@@ -275,11 +275,11 @@ def before_destroy_actions
   end
 
   def bust_cache_without_delay
-    Comments::BustCacheJob.perform_now(id)
+    Comments::BustCacheWorker.perform_async(id)
```

Comment:
`_without_delay` method should perform an action synchronously. Current change changes behavior.
This method is also not used a lot, I would use this pr as an opportunity to remove the method and replace its call with inline call like `Comments::BustCacheWorker.new(id).perform`.

Reply:
Sorry for having missed that! Thanks a lot @lightalloy!!!

Reply:
woops my bad, I (manually) did a blind search/replace of `perform_*` with `perform_async`, will fix and also add a covering spec to catch this